integrationtests/tests/tests.rs

```diff
@@ -366,14 +371,13 @@ async fn receive_lightning_payment_invalid_preimage() {
         .claim_incoming_contract(contract_id, rng())
         .await
         .unwrap();
-    fed.run_consensus_epochs(4).await; // 2 epochs to process contract, 2 to sweep ecash from contract
-
-    // TODO: disable background_fetch for this test and fetch the coins manually
```

Comment:
We should probably make an issue to this. It relates to the broader question for the CLN plugin of how to synchronize concurrent background fetcher, gateway webserver, and cln plugin.